### DISCUSSION TITLE
Template that should be removed in release changelog of gix

### DISCUSSION THREAD

**Discussion by NobodyXu** [GitoxideLabs/gitoxide#1158] [General]
In the second changelog of latest release of gix:

> Please enter the message for your patch. Lines starting with

**Comment by NobodyXu** [comment]
cc @Byron

**Reply by Byron** [reply]
Thanks for the hint! It's fixed in https://github.com/Byron/gitoxide/pull/1159 .

Apparently a line slipped into the commit message, which then was picked up for the changelog as well.

**Reply by NobodyXu** [reply]
Thanks!
